### ISSUE TITLE
Emunand/Rednand Setup Failure

### ISSUE THREAD

**Issue by annson24** [d0k3/GodMode9#21]
After formatting the SD card to have Emunand/Rednand Partition using GM9, it just goes back to the main menu. No options to clone Sysnand to Emunand/Rednand.

**Comment by d0k3** [comment]
Did you select the correct size for your EmuNAND? If you are on a N3DS, you need to select N3DS NAND size or 2GB (legacy size).

**Comment by annson24** [comment]
I believe so, let me try again and report back later.

**Comment by d0k3** [comment]
Okay, I've been able to reproduce this. Expect a fix shortly.

**Comment by d0k3** [comment]
Here's a testbuild that should have this fixed:
https://transfer.sh/fSP8N/godmode9-20170116-145449.zip

Can you try this and tell me if it works for you now?

**Comment by d0k3** [comment]
Tested it myself, fixed in:
https://github.com/d0k3/GodMode9/commit/92eb3901f3c3f43ab86b3ba14713dc9b869b6b3a

**Comment by annson24** [comment]
Thanks, it worked on me too.
